src/Config/user_local.props

```diff
@@ -6,5 +6,6 @@
 	<DYNAMOTESTAPI Condition=" '$(DYNAMOTESTAPI)' == '' ">$(SolutionDir)..\bin\$(Platform)\$(Configuration)</DYNAMOTESTAPI>
 	<OutputPath Condition=" '$(OutputPath)' == '' ">$(SolutionDir)..\bin\AnyCPU\$(Configuration)\$(REVIT_VERSION)</OutputPath>
 	<TestOutputPath Condition=" '$(TestOutputPath)' == '' ">$(OutputPath)</TestOutputPath>
+  <REVITAPI Condition=" '$(REVITAPI)' == '' ">$(SolutionDir)..\..\Revit\$(Configuration)x64</REVITAPI>
```

Comment:
hmm it looks like a relative path here. Is it because for you, Dynamo and Revit are in the same folder?

Reply:
@QilongTang Yes, on my system, my revit build and dynamoRevit build are on the same "level" so this gets me the location of my build to work with.  If I read cs.props correctly, if this is not true on a machine, it will be ignored due to the exist check in there... we could add it here as well, but seems a convenient thing to have in place for people.